Add changes to oidc docs in concepts section. (#2636)
wso2/docs-is: en/docs/references/concepts/authentication/id-token.md

`<header>.<body>.`

    ```
    eyJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJhbGljZSIsImlzcyI6Imh0dHBzOlwvXC9jMmlkLmNvbSIsImlhdCI6MTQxNjE1ODU0MX0
    ```

2. Signed ID token

    A sample of the **signed ID token** is shown below. It has three parts separated by `.` in the format of `<header>.<body>.<signature>`.

    ```
    eyJ0eXAiOiJKV1QiLA0KICJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhbGljZSIsImlzcyI6Imh0dHBzOlwvXC9jMmlkLmNvbSIsImlhdCI6MTQxNjE1ODU0MX0.iTf0eDBF-6-OlJwBNxCK3nqTUjwC71-KpqXVr21tlIQq4_ncoPODQxuxfzIEwl3Ko_Mkt030zJs-d36J4UCxVSU21hlMOscNbuVIgdnyWhVYzh_-v2SZGfye9GxAhKOWL-_xoZQCRF9fZ1j3dWleRqIcPBFHVeFseD_64PNemyg
    ```

    After decoding the header value `eyJ0eXAiOiJKV1QiLA0KICJhbGciOiJIUzI1NiJ9`  of the ID token it looks as follows.

    ```
    {"typ":"JWT",
    "alg":"HS256"}
    ```
    This header section can contain any parameter described in the [spec](https://tools.ietf.org/html/rfc7515#section-4.1).

    After decoding the payload value of the ID token, it looks as follows.  The payload can be an arbitrary sequence of octets, representing user attributes.

    ```
    {
      "sub": "alice",
      "iss": "https://c2id.com",
      "iat": 1416158541
    }
    ```

> !!! tip "Things to consider when issuing an ID token"

Reviewer: ```suggestion
!!! tip "Things to keep in mind when issuing an ID token"
```

>     1. **Not** to add any sensitive data to the token payload.

Reviewer: ```suggestion
    1. Do not add any sensitive data to the token payload.
```

>     2. To have a finite and short expiry time.

Reviewer: ```suggestion
    2. Have a finite and short expiry time.
```

    3. Sign/encrypt the JWT properly and validate it from the client side.

>     4. To include any additional optional parameters based on the use case.

Reviewer: ```suggestion
    4. Include any additional optional parameters based on the use case.
```
